ci-operator/config/openshift/velero/openshift-velero-oadp-1.4.yaml

```diff
@@ -33,7 +33,7 @@ tests:
   steps:
     test:
     - as: unit
-      commands: make -f Makefile.prow ci
+      commands: PATH="/go/bin:$PATH" make -f Makefile.prow ci
```

Comment:
Which image is that from?

I wonder if it's not easier to use one of the `go env` to ensure it's there ,because we can't be sure konveyor will always stay there:

https://github.com/konveyor/builder/blob/main/Dockerfile.ubi9#L13-L32

Reply:
From here https://github.com/openshift/ci-tools/blame/master/pkg/steps/source.go#L102

This is added to our image when our repo is configured to use imagestreamtag which openshift prow replaces `FROM` in our dockerfile. Not something specified in konveyor builder.

Reply:
image_stream_tag line 2 (which would be fine)

but as Tiger noted https://github.com/openshift/ci-tools/issues/4165 this is needed

Reply:
thanks !